superset-frontend/src/pages/ChartList/index.tsx

```diff
@@ -645,6 +647,7 @@ function ChartList(props: ChartListProps) {
           props.user,
         ),
         paginate: true,
+        width: 150,
```

Comment:
Owner contain people names which require more width.
```suggestion
```

Reply:
I can bump it back to 200px but was trying to save some more space to fit things in a single row. Arguably many of the dropdowns need more dropdown width than in the pop-down list (datasets, charts, dashboard names can be way beyond 200px). Luckily they all have a title prop which means hovering for a moment will show the full name. 

I think antd v5 will make it easier to have wider with in the popdown than in the trigger/input